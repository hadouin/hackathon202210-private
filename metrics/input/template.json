{
	"1": {
		"type": "templating_dlms",
		"arguments": {"filename_id":"templating_id_1","dt_start":"2022-08-16T13:35:00","dt_stop":"2022-08-16T13:35:00"}
	}
}