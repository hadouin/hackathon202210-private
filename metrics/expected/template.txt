1 ['ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404', 'ad8e9b26da6920bc2338c93eca6af404']
